### PR TITLE
Make styling of groups tab in the explore page use the Backstage theme

### PR DESCRIPTION
**Pull request by Fox32**
It the current style annoyed my a bit to much 😅 Looks like it was just the [example values from storybook ](https://backstage.io/storybook/?path=/story/data-display-dependencygraph--custom-nodes)😆 
I also changed the layout to be centered full screen. That way big diagram might be hard to read, but you can zoom in them. Previously you had to scroll down the page, which collides with the zooming functionality.

The system diagram has the same (default) look, but we aren't using it internally. We wrote a bigger graph plugin that could replace it once we made it open source.

**Before:**
image

image

**After:**

image

image

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by freben** [comment]
So I guess this makes it (especially the light mode) very low contrast, and a bit less snazzy :) Hm. Could it be worth picking harder contrast values in any case, whether we use colors or not?

**Comment by freben** [comment]
It's hard to convey taste based feedback, but to put it more into practical terms, what if these looked more like the outlined and main MUI buttons did at least? Where the latter indeed even implies color, typically

**Comment by Fox32** [comment]
> It's hard to convey taste based feedback, but to put it more into practical terms, what if these looked more like the outlined and main MUI buttons did at least? Where the latter indeed even implies color, typically

@freben Could be an option too, I will have a look. We could use primary color for the normal ones a the secondary color for the root.

But would you go with colored text too?

image

**Comment by Fox32** [comment]
Gave the colors an update, now it looks way more colorful 😆 But I like it - similar to the old style, but now themeable.

image
image

**Comment by Fox32** [comment]
Now includes the changes for the `<DependencyGraph>`

File: plugins/explore/src/components/GroupsExplorerContent/GroupsExplorerContent.tsx [hunk 1]
```diff
@@ -29,8 +38,10 @@ type GroupsExplorerContentProps = {
 export const GroupsExplorerContent = ({
   title,
 }: GroupsExplorerContentProps) => {
+  const classes = useStyles();
+
   return (
-    <Content noPadding>
+    <Content noPadding stretch className={classes.root}>
```
Comment (freben) [inline plugins/explore/src/components/GroupsExplorerContent/GroupsExplorerContent.tsx:44]: what does it mean to stretch AND have those styles?

**Review by freben — APPROVED** [review]
looks good to me!

File: plugins/explore/src/components/GroupsExplorerContent/GroupsExplorerContent.tsx (hunk 1 above)
Comment (Fox32) [reply plugins/explore/src/components/GroupsExplorerContent/GroupsExplorerContent.tsx:44]: Stretch only configures flex box for the elements inside `<Content/>`, the styles set the size of the content area itself. The max height is required as the SVG element likes to grow bigger then height. The whole layout thing isn't that nice to solve here… And I might missed some edge cases.

**Pull request merged**